plugins/repository-azure/licenses/azure-core-1.41.0.jar.sha1

```diff
@@ -0,0 +1 @@
+851e73f3856fc40c5dc96384ee3a3f3b126f8627
```

Comment:
https://repo1.maven.org/maven2/com/azure/azure-core/1.39.0/azure-core-1.39.0.jar.sha1
The actual sha1 `39765fb88a90174628b31ddf6ff9f8d63462e080`
@peterzhuamazon @saratvemulapalli @andrross